cypress/support/commands.js

```diff
@@ -0,0 +1,9 @@
+Cypress.Commands.add('visitUrl', (label) => {
+    cy.visit(label)
+})
+
+Cypress.Commands.add('checkTitleText',(text) => {
+    return cy.get('app-root > p').should(($text) => {
```

Comment:
Mucho mejor si lo pones como:
`cy.get('app-root > p').should('have.text','<texto a comprobar>')
cy.get('app-root > p').should('be.visible')`

De esta manera compruebas el texto que deseas que tenga y si está visible y no solo representado en el DOM